src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -201,6 +202,14 @@ struct BerdyOptions
     std::vector<std::string> jointOnWhichTheInternalWrenchIsMeasured;
 
     /**
+     * Name of the link which will be considered as a base frame.
```

Comment:
We are always discussing of "base link", so per consistency I would use "base link". here.